doc/api/process.md

```diff
@@ -1817,6 +1817,71 @@ a code.
 Specifying a code to [`process.exit(code)`][`process.exit()`] will override any
 previous setting of `process.exitCode`.
 
+## `process.getActiveResourcesInfo()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* Returns: {Array} of objects containing the `type`, `asyncId` and the
+  `triggerAsyncId` of the currently active requests, handles and timers.
+
+The `process.getActiveResourcesInfo()` method can be called at any moment and it
+would return an array of objects containing some information about all the
+resources that are currently keeping the event loop alive. This can be useful
+while debugging.
```

Comment:
```suggestion
The `process.getActiveResourcesInfo()` method returns an array of objects
containing information about the resources that are currently keeping the
event loop alive. This can be useful while debugging.
```

Reply:
Done